lib/logger.rb

```diff
@@ -340,12 +476,21 @@ def fatal!; self.level = FATAL; end
 
   #
   # :call-seq:
-  #   Logger.new(logdev, shift_age = 0, shift_size = 1048576)
-  #   Logger.new(logdev, shift_age = 'weekly')
-  #   Logger.new(logdev, level: :info)
-  #   Logger.new(logdev, progname: 'progname')
-  #   Logger.new(logdev, formatter: formatter)
-  #   Logger.new(logdev, datetime_format: '%Y-%m-%d %H:%M:%S')
+  #   Logger.new(logdev, shift_age = 0, shift_size = 1048576, **options)
+  #   Logger.new(logdev, shift_age = 'weekly', **options)
+  #
+  # With the single argument +logdev+,
+  # returns a new logger with all default options:
+  #
+  #   Logger.new('t.log') # => #<Logger:0x000001e685dc6ac8>
+  #
+  # Argument +logdev+ must be one of:
+  #
+  # - A string filepath: entries are to be written
+  #   to the file at that path.
+  # - An IO stream (typically +$stdout+, +$stderr+. or an open file):
```

Comment:
I think this is a typo:

```suggestion
  # - An IO stream (typically +$stdout+, +$stderr+, or an open file):
```